Xamarin.Forms.Controls/ControlGalleryPages/FlowDirectionGallery.cs

```diff
@@ -0,0 +1,429 @@
+﻿using System;
+using System.Linq;
+
+namespace Xamarin.Forms.Controls
+{
+
+	public class FlowDirectionGalleryLandingPage : ContentPage
+	{
+		FlowDirection DeviceDirection => Device.Info.CurrentFlowDirection;
```

Comment:
is `Device.Info.CurrentFlowDirection` the recommended way to access the FlowDirection?

if so, it causes 2 problems:
- Device.Info is attributed with `[EditorBrowsable(EditorBrowsableState.Never)]`
- it makes it hard to access from a `XAML` page. the FlowDirection should probably be exposed at the `Device` level, so one can do `<ContentPage FlowDirection="{x:Static Device.FlowDirection}" ... />` (`x:Static` syntax is `Type.Member` and doesn't support `Type.Member.Member`)

Reply:
I like it. Trying it out now...